kernel: bump 5.15 to 5.15.78

graysky2 opened the pull request:
All patches automatically rebased

Build system: x86_64
Build-tested: bcm2711/RPi4B, mt7622/RT3200
Run-tested: bcm2711/RPi4B, mt7622/RT3200

Signed-off-by: John Audia <therealgraysky@proton.me>

jospezial commented:
Merge-Konflikt in target/linux/ramips/patches-5.15/700-net-ethernet-mediatek-support-net-labels.patch
See also https://github.com/openwrt/openwrt/commit/04b8ad60fd92b375cf661a56afb853344c95ca99

graysky2 committed:
kernel: bump 5.15 to 5.15.78

All patches automatically rebased

Build system: x86_64
Build-tested: bcm2711/RPi4B, mt7622/RT3200
Run-tested: bcm2711/RPi4B, mt7622/RT3200

Signed-off-by: John Audia <therealgraysky@proton.me>

hauke commented:
Thank you for the patch, I applied it to master in 0252c0b1bce04d5312b71579582854b3badc3714.
